test/routes/contact.test.ts

```diff
@@ -19,7 +18,7 @@ describe('/contact', () => {
 
   beforeEach(() => {
     roService = new RoService(undefined, undefined)
-    mocked(roService).findResponsibleOfficer.mockResolvedValue({
+    ;(jest as any).mocked(roService).findResponsibleOfficer.mockResolvedValue({
```

Comment:
this is gnarly - need to wait until jest type defs are updated to match latest release